vendor/maplibre-native-base/.github/workflows/ci.yml

```diff
@@ -25,6 +25,8 @@ jobs:
     runs-on: ${{ matrix.os }}
```

Comment:
Oh this used to be a submodule. Need to move these workflows. Let me take a not of that.